remote-messaging/remote-messaging-impl/src/main/java/com/duckduckgo/remote/messaging/impl/ui/RemoteMessageModalSurfaceEvaluator.kt

```diff
@@ -50,6 +52,7 @@ interface RemoteMessageModalSurfaceEvaluator
 class RemoteMessageModalSurfaceEvaluatorImpl @Inject constructor(
     @AppCoroutineScope private val appCoroutineScope: CoroutineScope,
     private val remoteMessagingRepository: RemoteMessagingRepository,
+    private val settingsDataStore: SettingsDataStore,
```

Comment:
After moving this class out of app module I cannot use the SettingsDataStore anymore.